### PR TITLE
The Data Carrier (New) - The Grand Arcadia

### PR DESCRIPTION
**Pull request by AjexRose**
## About the PR

after finally getting off my ass, I decided to finally finish this, this PR adds a new bluespace event known as the "Arcadia Industries Data Carrier" A bluespace event within a fully operational shuttle, filled with lots of loot and entities to fight though. Also includes changes to the Arcadia Industries that were added awhile ago, mainly replacing their blood with oil, and making the Mobile Blaster Units space proof. 

## Why / Balance

Gives Arcadia Industries more visibility then just a vending machine. 
Edit: Also forgot to mention most of the crates are simple random low value spawners.
The high amount of loot within is countered by the fact that the shuttle is simply bursting from the seems with enemies, all ready to pounce at intruders.
The main focus is the R&D loot, which there isnt much attention for as of now in the Frontier. 
## Technical details

N/A
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Frontier Station 1_23_2024 2_34_38 PM
Frontier Station 1_23_2024 2_31_25 PM

## Breaking changes

**Changelog**

:cl: AjexRose
- add: Added a new bluespace event: The Data Carrier, with lots of loot, enemies, and a small focus on R&D
- tweak: Arcadia Industries mobs have been slightly reworked to no longer bleed blood.
- tweak: Fixed the ArcadiaDrobe's vendor sprite

### CODE REVIEW

File: Resources/Prototypes/_NF/Events/events.yml [hunk 1]
```diff
@@ -140,3 +140,22 @@
...
+- type: entity
+  id: BluespaceArcIndDataCarrier
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    startAnnouncement: station-event-bluespace-ArcIndDataCarrier-start-announcement
+    startAudio:
+      path: /Audio/Announcements/attention.ogg
+    endAnnouncement: station-event-bluespace-ArcIndDataCarrier-end-announcement
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/_NF/Events/events.yml:153]: Same here, prefer to use "station-event-bluespace-ship-end-announcement"

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Remove Random Magic Book Spawner (Or at least use the safe one)

Recomendation/Notes:
* Wizard hardsuit seem compleatly out of place (but also because there was plan to put those in more wizard/alien looking asteroids)
* Atmos is ehh, started then gone?
* Floor need decals in my opinion a bit.
* Does your NPCs has the salvage component? if no they need it so they despawn when the ship despawn.
* Again make the ship name/annoncement to the default/random.

File: Resources/Maps/_NF/Bluespace/DataCarrier.yml [hunk 2]
```diff
@@ -0,0 +1,9938 @@
...
+  106: FloorWhiteMono
+  111: FloorWood
+  113: Lattice
+  114: Plating
+entities:
+- proto: ""
+  entities:
+  - uid: 1
+    components:
+    - name: AI-DC01-003
```
[the remark this answers is not in the document]
Comment (AjexRose) [reply Resources/Maps/_NF/Bluespace/DataCarrier.yml]: Done

**Comment by AjexRose** [comment]
After testing it via addgamerule and endgamerule, the mobs despawn without the salvage component when the event ends. Either that or they already have it somewhere.

**Comment by AjexRose** [comment]
also the spell book was set to safe, and the wizard hardsuit was replaced by syndicate hardsuit

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl [hunk 3]
```diff
@@ -9,3 +9,4 @@ station-event-bluespace-asteroid-end-announcement = In compliance with NanoTrase
 
 station-event-bluespace-ship-start-announcement = We have detected an unusual FTL signature - long range scans indicate an unknown ship. NanoTrasen cannot confirm safety for prospectors within its vicinity, be advised.
 station-event-bluespace-ship-end-announcement = In compliance with NanoTrasen FTL traffic patterns, the unknown ship has been dissipated to ensure non-collision.
+
```
Comment (FoxxoTrystan) [inline Resources/Locale/en-US/_NF/bluespace-events/events.ftl:12]: Hmmm Space!

**Review by FoxxoTrystan — COMMENTED** [review]
All good, tho the Syndicate Hardsuit is an Elite one and seem a bit too easy to get in my opinion but overall il say generally its fine.

**Review by FoxxoTrystan — APPROVED** [review]

**Comment by AjexRose** [comment]
Updated to have more decals, removed the omega soap, replaced the syndicate hardsuit with AME fuel, switched power from damaged RTGs to regular RTGs, replaced the unused atmos space with a space for portable scrubbers.

**Comment by Cheackraze** [comment]
> Updated to have more decals, removed the omega soap, replaced the syndicate hardsuit with AME fuel, switched power from damaged RTGs to regular RTGs, replaced the unused atmos space with a space for portable scrubbers. Also changed the file name to match current naming standards

was just looking to do some map reviews and such. Do you have any pictures of the new decaling? it was going to be my only critique of the map itself. Otherwise, I can help update this to the latest build and get this in game, however the vending machine sprite needs fixing desperately as well, if you wouldnt mind taking a look at cleaning that one up.

**Comment by Cheackraze** [comment]
just tested it out a few times, the map itself looks nice, especially with the added decals. I'm only a little concerned that it will get powergamed everytime and might in the future re-arrange what items are stored in each of your vault areas. If we could just get that vending machine sprite cleaned up i'd say its good to merge

**Comment by AjexRose** [comment]
Just got home from uni, so apologies for not getting back to you sooner
> just tested it out a few times, the map itself looks nice, especially with the added decals. I'm only a little concerned that it will get powergamed everytime
Most items are simple fill-in substitutes until I get more faction unique items made.
> If we could just get that vending machine sprite cleaned up i'd say its good to merge
I'll start to work on that ASAP

**Comment by AjexRose** [comment]
vending machine's display sprite has been fixed at last!

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**